src/components/views/rooms/EventTile.js

```diff
@@ -472,6 +475,17 @@ module.exports = withMatrixClient(React.createClass({
         return this.refs.replyThread;
     },
 
+    getReactions() {
+        if (
+            !SettingsStore.isFeatureEnabled("feature_reactions") ||
+            !this.props.getRelationsForEvent
```

Comment:
I'd recommend flipping the order of these calls for performance reasons: SettingsStore is slow.

Reply:
Okay, done! (Just how slow is it...?)